### PR TITLE
Register Drivers and Operators

### PR DESCRIPTION
**Pull request by Egide138**
### What does this PR do?
• Register Driver and operators to Phantom
### Description of Task to be completed?
• Authenticated users are able to  register operators and drivers.
• Appropriate error messages are displayed on unsuccessful attempts
• Skeleton UIs/screen are/is shown when the page loads/is loading.
• The app layouts are displayed
• The page is responsive.
### How should this be manually tested?
• checkout this branch
• run npm install
•  register  a user
• click add new operator
### What are the relevant trello id?
SLJq5tgX
Labels
work in progress

### CODE REVIEW

**Review by mansurissa — DISMISSED** [review]
LGTM!

**Review by GuevaraMarius — DISMISSED** [review]

**Review by 41hulk — DISMISSED** [review]
LGTM

File: public/locales/en/translation.js [hunk 1]
```diff
@@ -113,6 +113,62 @@ export default {
...
+    3: 'operator',
+    4: 'Driver',
+    5: 'Route',
+    6: 'Settings',
+    7: 'New operator',
+    8: 'All Operators',
+    9: 'No',
+    10: 'First Name',
+    11: 'Last Name',
+    12: 'Natinal ID',
```
Comment (raymond42) [inline public/locales/en/translation.js]: Typo here, it should be `National id`

**Review by raymond42 — CHANGES_REQUESTED** [review]
Good work @Egide138. Only a few comments to work on.
1. There is a typo on the National id label
2. It could be better if close the modal after registering a driver or an operator
3. Let's have a warning before deleting an operator or a driver, i.e display an alert in form of a model asking him if he wants to continue with the action.

**Review by mansurissa — DISMISSED** [review]
Good Job

**Review by KundwaBruno — DISMISSED** [review]

**Review by GuevaraMarius — DISMISSED** [review]

**Review by raymond42 — APPROVED** [review]

**Review by 41hulk — APPROVED** [review]

**Review by SIBONIYO — APPROVED** [review]
LGTM

**Review by mansurissa — APPROVED** [review]
LGTM!

**Pull request merged**